salt/pythontest/config/vsftpd.conf

```diff
@@ -0,0 +1,45 @@
+ftpd_banner=Welcome to pythontest FTP server
+
+# Run vsftp as a standalone server
+listen=YES
+
+# Allow anonymous FTP. (This is what the test suite uses)
+anonymous_enable=YES
```

Comment:
can we also set an `anon_root` to force anonymous access into the appropriate directory?